scripts/init.js

```diff
@@ -64,6 +64,9 @@ $(document).ready(function() {
 		.bind([ 'command+a', 'ctrl+a' ], function() {
 			if (visible.album() && basicModal.visible()===false)       { multiselect.selectAll(); return false }
 			else if (visible.albums() && basicModal.visible()===false) { multiselect.selectAll(); return false }
+		})
+		.bind([ '0' ], function() {
+			if(visible.photo()) { settings.setOverlayType('exif')}
```

Comment:
I would use another `o` to switch between `date` => `exif` => `desc` => `date` => ... :)
I think you binded `0` instead of `o`. ( `o` like Overlay ;) )

Also that is very wrong. You are manipulating server settings with this operation. Server settings should be changed in the Settings tab.

Basically you need to create a function that change locally  `lychee.image_overlay_type` to the next value and that update visually the overlay.